release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -804,6 +804,34 @@ With this release, the etcd tuning parameters can be set to values that optimize
 * `Standard`
 * `Slower`
 
+[discrete]
+[id="ocp-4-16-unauthenticated-users-cluster-role-bindings_{context}"]
+=== Unauthenticated users were removed from some cluster roles
+
+With this release, unauthenticated users no longer have access to specific cluster roles that are necessary for certain feature sets. Before {product-title} {product-version} unauthenticated users could access certain cluster roles. Changing this access for unauthenticated users adds an additional layer of security and should only be enabled when necessary. This change is for new clusters; previous clusters are not affected.
+
+There are use cases where you might want to give access to unauthenticated users for specific cluster roles. To grant unauthenticated users access to specific cluster roles that are necessary for certain features, see xref:../modules/unauthenticated-users-cluster-role-binding.adoc#modules-unauth-cluster-role-bindings[Adding unauthenticated groups to cluster role bindings].
+
+[IMPORTANT]
+====
+Always verify compliance with your organization's security standards when modifying unauthenticated access.
+====
+
+[discrete]
+[id="ocp-4-16-unauthenticated-users-system-webhook-role-binding_{context}"]
+=== Unauthenticated users were removed from the `system:webhook` role binding
+
+With this release, unauthenticated users no longer have access to the `system:webhook` role binding. Before {product-title} {product-version}, unauthenticated users could access the `system:webhook` role binding. Changing this access for unauthenticated users adds an additional layer of security and should only be enabled by users when necessary. This change is for new clusters; previous clusters are not affected.
+
+There are use cases where you might want to allow unauthenticated users the `system:webhook` role binding for specific namespaces. The `system:webhook` cluster role allows users to trigger builds from external systems that do not use {product-title} authentication  mechanisms, such as GitHub, GitLab, and Bitbucket. Cluster admins can grant unauthenticated users access to the `system:webhook` role binding to facilitate this use case.
+
+[IMPORTANT]
+====
+Always verify compliance with your organization's security standards when modifying unauthenticated access.
+====
+
+To grant unauthenticated users access to the `system:webhook` role binding in specific namespaces, see xref:../../triggering-builds-build-hooks.adoc#unauthenticated-users-system-webhook_triggering-builds-build-hooks[Adding unauthenticated users to the system:webhook role binding].
+
```

Comment:
Should this be moved to the "Build" section of "New features and enhancements"? I'd call AUTH-509 a security enhancement, and moving it in the "Build" section places the impact front and center. I don't think any other component in OCP uses the "system:webhook" role directly.

Reply:
@adambkaplan I can move it there if you think that's a better place for customers to find it.

Reply:
I think that will help customers discover the change more easily - thanks!

Reply:
@adambkaplan I've moved it. The preview takes a few minutes to update with the change if you want to take a look.